script/src/verify.rs

```diff
@@ -566,14 +585,16 @@ impl<DL: CellDataProvider + HeaderProvider + ExtensionProvider + Send + Sync + C
 
     fn build_state(
         &self,
-        vm: Option<ResumableMachine>,
+        vms: Vec<ResumableMachine>,
```

Comment:
build_state is a pure fn, suggest to remove `self` or use `TransactionState::new` directly

Reply:
Done